src/richie/apps/courses/models/course.py

```diff
@@ -111,7 +113,23 @@ class Course(BasePageExtension):
     page that presents the course.
     """
 
-    PAGE = COURSES_PAGE
+    effort = EffortField(
+        intervals=defaults.INTERVALS,
+        default_effort_interval=defaults.DEFAULT_EFFORT_INTERVAL,
+        default_reference_interval=defaults.DEFAULT_REFERENCE_INTERVAL,
+        max_length=80,
+        blank=True,
+        null=True,
+    )
+    length = CompositeDurationField(
```

Comment:
What about `duration` instead?

Reply:
I used the same name as edX... I'm not english native but I also prefer duration...